prediction-polls/frontend/src/Pages/Auth/SignIn/index.jsx

```diff
@@ -101,9 +107,12 @@ function SignIn() {
       if (response.status === 200) {
```

Comment:
You are sending your request to local host and checking the wrong status code I guess. Will you change them later?

Reply:
Changed them now, sir!

Reply:
URL seems nice now but do you get status 200 for successful request?